### ISSUE TITLE
Cannot load modules via web interface

### ISSUE THREAD

**Issue by Agent-Isai** [ProjectFirrre/zncstrap#16]
I tried loading network modules like sasl via the web interface but I got:
Error!
Network name is a required argument
Then I tried adding a network and loading modules and I again got
Error!
Network name is a required argument
Even though I put a network name, any idea why?

**Comment by sprybear** [comment]
Hi, Agent-Isai,

Which version of ZNC are you on and which zncstrap branch are you making use of (the 'master' or 'dev'), mind asking?

**Comment by Mikaela** [comment]
I have seen multiple questions about this on \#znc at freenode with ZNC 1.6.0.

**Comment by sprybear** [comment]
Thank you for letting me know of this, Mikaela.

I did get a ton of private messages actually in regards to this. For the most, users have been using the wrong branch for zncstrap.

Currently, the 'master' branch is specifically for ZNC v1.4, while the 'dev' branch is for v1.5/1.6 and above. Both branches varies.

In the next month or so, I will make some adjustments to the branches. E.g: Swap the current 'master' branch with the 'dev' branch; and the current 'master' branch to 'legacy' etc.

Had some downtime a few minutes ago, so I decided to do a fresh installation of ZNC v1.6 and try to replicate the issue with zncstrap. Only when I used zncstrap 'master' branch version on ZNC v1.6 setup did I get the "Module not loaded ..." error message, which is the wrong branch. 

However, I am not going to throw it out the window. Well, it would be lame of me to do so. -- If the issue/problem still persists using ZNC v1.6 (current stable version) on zncstrap 'dev' branch version, I'd suggest users to report the issue via GitHub. Preferably, attach screenshot(s) too.

Will make an effort to lurk around #znc even more. *winks*
